model/testresult/testresult.go

```diff
@@ -29,19 +29,32 @@ type TaskTestResultsStats struct {
 
 // TestResult represents a single test result from an Evergreen task run.
 type TestResult struct {
-	TaskID          string    `json:"task_id" bson:"task_id"`
-	Execution       int       `json:"execution" bson:"execution"`
-	TestName        string    `json:"test_name" bson:"test_name"`
-	DisplayTestName string    `json:"display_test_name" bson:"display_test_name"`
-	GroupID         string    `json:"group_id" bson:"group_id"`
-	Status          string    `json:"status" bson:"status"`
-	BaseStatus      string    `json:"base_status" bson:"base_status"`
-	LogTestName     string    `json:"log_test_name" bson:"log_test_name"`
-	LogURL          string    `json:"log_url" bson:"log_url"`
-	RawLogURL       string    `json:"raw_log_url" bson:"raw_log_url"`
-	LineNum         int       `json:"line_num" bson:"line_num"`
-	TestStartTime   time.Time `json:"test_start_time" bson:"test_start_time"`
-	TestEndTime     time.Time `json:"test_end_time" bson:"test_end_time"`
+	TaskID          string       `json:"task_id" bson:"task_id"`
```

Comment:
Are we intentionally making all the BSON tags not `omitempty` for a reason? I thought the recommendation from the Mongo driver was to use `omitempty` where possible.

Reply:
I can add omitempty tags but we only use Mongo for local development.

Reply:
So just to clarify - the BSON tags are only relevant for local development and prod never stores these?